src/pages/publication/PublicProfile.tsx

```diff
@@ -59,22 +61,29 @@ const PublicProfile: FC = () => {
       <StyledUserInfo>
         <Card>
           <Heading>{user.name}</Heading>
-          <LabelTextLine dataTestId="profile-email" label={t('common:email')} text={user.email} />
+          <LabelTextLine label={t('heading.organizations')}>
+            {isLoadingMultiple ? (
+              <CircularProgress color="inherit" size={20} />
+            ) : (
+              units?.map((unit) => <NormalText key={unit.id}>{unit.name}</NormalText>)
```

Comment:
hvis vi bare viser top-level vil det bli litt rart ut om man feks er tilknyttet flere fakultet ved NTNU. Ser det ikke er mer å hente på wireframes da, så det er sikkert greit for nå

Reply:
ja, jeg spurte Terje, og han sa det så greit ut foreløpig. men hvis vi trenger underinstitusjoner så får vi bare endre på det i ettertid tenker jeg